reference/docs-conceptual/Crescendo/get-started/choose-command-line-tool.md

```diff
@@ -0,0 +1,73 @@
+---
+description: How to select a command-line tool to amplify using Crescendo.
+ms.date: 03/08/2022
+title: Choosing a command-line tool to amplify
+---
+# Choosing the command-line tool for Crescendo
+
+Developing a cmdlet with Crescendo is a rapid, reliable way to _amplify_ a command-line tool to
+produce a cmdlet-like experience. Many times, the tool you need to work with is directly tied to the
+technology you are attempting to automate. This makes the tool choice simple. However, not all tools
+work the same way. To optimize your investment when developing a cmdlet, consider the following
+criteria:
+
+## Command-line tools that make good candidates
+
+- **The original tool is difficult to use.**
+
+  If the tool is simple to use and provides the information you need, there is no need to create a
+  cmdlet. However, this isn't always the case with all command-line tools. Many command-line tools
+  have their own unique syntax, parameters, and output that make it difficult for inexperienced
+  administrators to use the command. Converting the command into a cmdlet provides all the benefits
+  of cmdlet discovery, syntax consistency, and structured output as objects.
+
+- **The command-line tool output is difficult to use in automation.**
+
+  Command-line tools output their information to the screen as string data. This is not the
+  structured data (objects) that PowerShell expects in the pipeline, which prevents you from using
+  PowerShell cmdlets, such as `Where-Object` and `ForEach-Object`. Crescendo assists you with
+  _amplifying_ the command-line tool experience to a cmdlet so that the tool can participate in the
+  PowerShell pipeline.
+
+- **The command-line tool doesn't provide adequate help.**
+
+  Command-line tools that lack help information can be difficult to use. Cmdlets can have
+  comprehensive help information that includes details on parameters, descriptions, and examples.
+  Crescendo gives you the ability to create the missing help for your amplified commands.
+
+## Command-line tools that make bad candidates
+
+- **Does a cmdlet already exist?**
+
+  PowerShell has a well-established ecosystem of modules that extend the built-in capabilities of
+  PowerShell. Before deciding to amplify a command-line tool with Crescendo, first check to see if a
+  cmdlet already exists that performs your intended goal. For example, wrapping the Windows command
+  `ipconfig.exe` to get the current IP address is a poor choice because the module **NetAdapter**
+  contains `Get-NetIPConfiguration`, which provides the same information with structured output.
+
+- **Are there better ways?**
+
+  Often the information you need for automation can be obtained quicker than making a cmdlet.
+  PowerShell lets you access other frameworks and libraries, such as WMI and .NET. Instead of
+  investing time creating a cmdlet, you might be able to get the information more quickly using one
```

Comment:
~~This implies to me as a naive reader that the effort is in _cmdlet development_ as opposed to writing a configuration file and (potentially) parser script. I'm okay if we're using _cmdlet development_ in these documents as shorthand for this, just want to be sure.~~

Edit: Stricken in favor of [this comment]() but left for clarity (and in case the decision is made to update terminology).

```suggestion
  investing time amplifying a command-line tool, you might be able to get the information more quickly using one
```